source/manual/logit.html.md

```diff
@@ -0,0 +1,63 @@
+---
+owner_slack: "#2ndline"
+title: How to use Logit for GOV.UK
+section: Logging
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2017-10-02
+review_in: 6 months
+---
+
+## How GOV.UK use Logit
+
+GOV.UK use [Logit](https://logit.io) to provide our [ELK Stack](https://www.elastic.co/webinars/introduction-elk-stack).
+
+This is in line with [The GDS Way](https://gds-way.cloudapps.digital/) guidance
+on [logging](https://gds-way.cloudapps.digital/standards/logging.html).
+
+## Logging in to Logit
+
+We use a Google App to authenticate users with Logit. Login to your GDS Google account in Chrome
+(for example, when in https://mail.google.com), and click on the 9 small dots at the top
+right of the Chrome browser.
+
+Go to the bottom of the selection of apps and you should find one for "Logit".
+
+Clicking on this should log you in.
+
+### Adding your user to the right team
+
+After you have logged in, if you are unable to see any "stacks", please speak to
+a GOV.UK Logit administrator.
```

Comment:
How can people find a GOV.UK Logit Administrator?

Reply:
I'll update it to be more specific ie "your tech lead"